Theodore Natoli
============

<!-- contact info center justified -->
> [ted.e.natoli@gmail.com](mailto:ted.e.natoli@gmail.com) • (857)-498-1946 • 109 Green St. Wakefield, MA 01880

> [Github](https://github.com/tnat1031) • [Research Gate](https://www.researchgate.net/profile/Ted_Natoli2) • [LinkedIn](https://www.linkedin.com/in/ted-natoli-compbio)


<!-- quote or summary statement set off by horizontal lines -->

----

>  I am an experienced data scientist seeking to utilize and improve my computational skills to help advance biomedical research.

----



Selected Publications
----------

* Subramanian, Aravind, Rajiv Narayan, Steven M. Corsello, David D. Peck, **Ted E. Natoli,** Xiaodong Lu, Joshua Gould, et al. 2017. "A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles." Cell 171 (6): 1437–52.e17.

* Amy Li, Xiaodong Lu, **Ted Natoli,** Joshua Bittker, Nisha S. Sipes, Aravind Subramanian, Scott Auerbach, David H. Sherr, and Stefano Monti. 2019. "The Carcinogenome Project: In Vitro Gene Expression Profiling of Chemical Perturbations to Predict Long-Term Carcinogenicity." Environmental Health Perspectives 127 (4).

* Smith, Ian, Peyton G. Greenside, **Ted Natoli,** David L. Lahr, David Wadden, Itay Tirosh, Rajiv Narayan, et al. 2017. “Evaluation of RNAi and CRISPR Technologies by Large-Scale Gene Expression Profiling in the Connectivity Map.” PLoS Biology 15 (11): e2003213.

* Berger, Alice H., Angela N. Brooks, Xiaoyun Wu, Yashaswi Shrestha, Candace Chouinard, Federica Piccioni, Mukta Bagul, et al. 2016. "High-Throughput Phenotyping of Lung Cancer Somatic Mutations." Cancer Cell 30 (2): 214–28.


Professional Experience
----------

2012-present
:   **Computational Biologist** • *The Broad Institute in Cambridge, MA*

    *Previously:* Associate Computational Biologist, Research Associate

    As a member of the Connectivity Map (CMap) group within the Broad’s Cancer Program, I process, analyze, and aid others in utilizing data from the L1000 high-throughput gene expression profiling assay. Responsibilities and achievements include:

    *  Lead and assist analysis efforts on internal and collaborative projects with academia and pharma in areas such as drug repurposing, drug discovery, and hit characterization. Experienced at presenting analysis methods and results to scientific audiences in various domains.

    * Develop analysis methods for integrating L1000 data with other high-throughput readouts such as cell viability and morphology.

    *  Advance CMap software development projects by acting as computational liaison for software engineers.

    *  Accelerate L1000 assay development by assisting the data generation team with ad-hoc analyses.

    * Improved efficiency and reliability of the L1000 analytical pipeline by writing utilities in Python and utilizing automated scheduling programs such as cron and Jenkins.

    *  Improved L1000 data tracking and organization by implementing and maintaining a LIMS database and web-based user interface to assist with managing the data production queue, tracking data generation batches, and easily viewing QC reports. Technology stack includes MySQL, Node.js and HTML/CSS/Javascript.


    * Develop and maintain library of CMap R code. 

    * Lead technical support and outreach efforts within the CMap user community.

    * Mentor and advise junior analysts and new group members.


<!-- split out RA position at Broad? -->

2009-2012
:    **Bioinformatics Scientist** • *Ligon Discovery in Cambridge, MA*

    *Previously:* Research Associate

    Lead bioinformatics efforts for drug discovery in a high throughput screening (HTS) environment by processing, managing, analyzing, and presenting data from small molecule microarray (SMM) and other biological assays. Specifically:

    * Increased SMM data analysis efficiency by building and implementing a pipeline using Perl and R. 
    
    *  Designed, built, and administered a MySQL LIMS database.
    
    * Implemented and ran SMM hit-calling analyses using R.
    
    * Managed and executed SMM projects for clients according to specified timelines and presented corresponding analysis results.



2007-2009
:   **Biologist and Techincal Support Engineer** • *Hamilton Thorne, Inc. in Beverly, MA*

    * Progressed product and application development by independently conducting biological R&D experiments. 
    
    * Improved in-house technical support capabilities and reduced volume of service calls by training repair technicians and customers on proprietary Computer Assisted Semen Analysis (CASA) hardware and software and the applications thereof.
    
    * Increased technical support efficiency and ensured product quality by performing mechanical, electrical and software validation, maintenance, and testing of CASA systems. 



2007
:   **Imaging Assistant** • *Perceptive Informatics (PAREXEL) in Waltham, MA*

    * Developed and coordinate clinical study-related activities in compliance with FDA regulatory requirements.
    
    * Used computer systems to convert medical image data to digital form.

    * Maintained a database of medical images and associated results.



Education
---------

2014 
:   **ALM, Biotechnology with Concentration in Bioinformatics**; Harvard Extension School (Cambridge, MA)

    *Thesis:* Implemented a full-stack web application for visualizing and analyzing CMap query results as a network graph.

2007
:   **BS, Biomedical Engineering**; Boston University (Boston, MA)



Technical Experience
--------------------

Programming Languages
:   **R:** Over 8 years of experience using R for exploratory data analysis, data visualization, and analysis workflows. Proficient in many packages including [data.table], [ggplot2], [shiny], and others. Author of the [cmapR](https://github.com/cmap/cmapR) package for manipulating annotated gene expression matrices.

:   **Python:** Over 4 years of experience using Python for data wrangling and writing scripts for web scraping and data processing pipeline automtation.

:   Basic knowledge of **Matlab**, **Javascript** including **Node.js**, **HTML/CSS**

Databases
:    **MySQL**, **MongoDB**

Cloud
:    **AWS** EC2 and S3, **Google** BigQuery

Other
:    **Git**, **Tableau**


----


