Theodore Natoli
============

<!-- contact info center justified -->
> [ted.e.natoli@gmail.com](mailto:ted.e.natoli@gmail.com) • (857)-498-1946 • Wakefield, MA 01880

> [Github](https://github.com/tnat1031) • [Research Gate](https://www.researchgate.net/profile/Ted_Natoli2) • [LinkedIn](https://www.linkedin.com/in/ted-natoli-compbio)


<!-- quote or summary statement set off by horizontal lines -->

----

>  Experienced computational biologist/bioinformatician/data engineer and team leader seeking an opportunity to advance biomedical research.

----


Professional Experience
----------

2022-2023
:   **Associate Director, Bioinformatics** • *ImmuneID in Waltham, MA*

    Lead bioinformatics efforts to help ImmuneID maximally leverage its data across high-throughput screening (HTS), target validation, and immunology.

    * Manage small team of data scientists, data engineers, and third party software development firm.

    * Serve as primary liaison between lab scientists and Bioinformatics team.

    * Develop and implement HTS data processing pipelines and data storage infrastructure on AWS and manage associated budget.

    * Develop QC metrics and data visualization tools for HTS data.

    * Author code libraries in R and Python, pipelines in Nextflow.

    * Design and implement LIMS database, including front-end using R Shiny, and associated ETL processes to synchronize multiple disparate data stores.


2012-2021
:   **Senior Computational Biologist and Group Leader** • *The Broad Institute in Cambridge, MA*

    *Previously:* Computational Biologist, Associate Computational Biologist, Research Associate

    As a member of the Connectivity Map [(CMap)](http://clue.io) and [PRISM](http:theprismlab.org) team within the Broad’s Cancer Program, I served as a lead data analyst and managed a team of computational biologists and software engineers on biomedical research projects utilizing data from the high-throughput [L1000 gene expression](https://clue.io/connectopedia/what_is_l1000) and [PRISM cell viability](http:theprismlab.org) assays. Responsibilities and achievements included:

    * Managed a team of analysts and software engineers around analysis strategy and execution, data infrastructure, and team and personal goal-setting.

    * Served as a mentor in Broad's Cancer Program 'Boot Camp' to help on-board new trainees and familiarize them with some of the institute's computational resources, including CMap and PRISM.

    * Led analysis strategy and execution and aid in experimental design for internal and collaborative projects with academia and pharma in areas such as drug repurposing, drug discovery, and hit characterization.

    * Led efforts to organize and release the updated CMap L1000 reference dataset containing over 3M gene expression profiles, along with accompanying software to enable data access.

    * Effectively visualized complex data and presented analysis methods and results to scientific audiences in various domains.

    * Developed analysis methods for integrating L1000 data with other high-throughput readouts such as cell viability and cell morphology.

    * Led efforts to structure and run crowdsourcing competitions which have resulted in improvements to 3 core CMap algorithms. 

    * Advanced CMap software development projects by acting as computational liaison for software engineers.

    * Developed and maintained a library of [CMap R code](http://bioconductor.org/packages/3.11/bioc/html/cmapR.html).

    * Improved efficiency and reliability of the L1000 data processing pipeline by writing utilities in Python and leveraging automated scheduling utilities such as cron and Jenkins.

    * Improved L1000 data tracking and organization by implementing a LIMS database and web-based user interface to assist with managing the data production queue, tracking data generation batches, and easily viewing QC reports. 



<!-- split out RA position at Broad? -->

2009-2012
:    **Bioinformatics Scientist** • *Ligon Discovery in Cambridge, MA*

    *Previously:* Research Associate

    Led bioinformatics efforts for drug discovery in a HTS setting by processing, managing, analyzing, and presenting data from small molecule microarray (SMM) and other biological assays. Specifically:

    * Increased SMM data analysis efficiency by building and implementing a pipeline using Perl and R. 
    
    * Designed, built, and administered a MySQL LIMS database.
    
    * Implemented and ran SMM hit-calling analyses using R.
    
    * Managed and executed SMM projects for clients according to specified timelines and presented corresponding analysis results.



2007-2009
:   **Biologist and Techincal Support Engineer** • *Hamilton Thorne, Inc. in Beverly, MA*

    * Independently conducted biological R&D experiments. 
    
    * Trained repair technicians and customers on proprietary Computer Assisted Semen Analysis (CASA) hardware and software and the applications thereof.
    
    * Performed mechanical, electrical and software validation, maintenance, and testing of CASA systems. 



2007
:   **Imaging Assistant** • *Perceptive Informatics (PAREXEL) in Waltham, MA*

    * Developed and coordinated clinical study-related activities in compliance with FDA regulatory requirements.
    
    * Used computer systems to convert medical image data to digital form.

    * Maintained a database of medical images and associated results.


Consulting
----------

2019
:   **Diamond Age Data Science** • *in Somerville, MA*

    Served as a consultant and advisor for a Diamond Age Data Science client on an analysis project involving public CMap data.


2016
:   **Navitor Pharmaceuticals** • *in Cambridge, MA*

    Performed differential expression analysis on microarray data to generate gene expression signatures of compounds.





Education
---------

2014 
:   **ALM, Biotechnology with Concentration in Bioinformatics**; Harvard Extension School (Cambridge, MA)

    *Thesis:* Implemented a full-stack web application for visualizing and analyzing CMap query results as a network graph.

2007
:   **BS, Biomedical Engineering**; Boston University (Boston, MA)



Technical Experience
--------------------

<!-- breaks at end of line add a litte breathing room around sections -->

Data Types
:    Gene expression ([L1000](https://clue.io/connectopedia/what_is_l1000), RNAseq), cell viability ([PRISM](https://depmap.org/portal/prism/), CTG), cell morphology ([cell painting](https://www.ncbi.nlm.nih.gov/pubmed/27560178))<br></br>


Resources
:    CMap, GEO, CCLE, DepMap, TGCA, GTEx, MSigDB, UniProt<br></br>

Analysis
:    Clustering, regression, inference, dimensionality reduction, feature selection, visualization<br></br>

OS
:    Linux, Mac, Windows<br></br>

Software Tools
:    Nextflow, GSEA, RSEM, Bowtie, Samtools, Benchling<br></br>

Programming Languages
:   **R:** Over 10 years of experience using R for exploratory data analysis, data visualization, and analysis workflows. Proficient in many packages/resources including [Bioconductor](https://www.bioconductor.org/), [data.table](https://cran.r-project.org/web/packages/data.table/vignettes/datatable-intro.html), [ggplot2](https://ggplot2.tidyverse.org/), [shiny](https://shiny.rstudio.com/), and others. Author of the [cmapR](http://bioconductor.org/packages/3.11/bioc/html/cmapR.html) package for manipulating annotated gene expression matrices.<br></br>

:   **Python:** Over 8 years of experience using Python for data wrangling and writing scripts for web scraping and data processing pipeline automation. Familiar with the [pandas](https://pandas.pydata.org/) and [numpy](https://www.numpy.org/) packages for data analysis and the [conda](https://docs.conda.io/en/latest/) package management system.<br></br>

:   **SQL:** Over 6 years of experience, largely with MySQL

:   Basic knowledge of **Matlab**, **Javascript** including **Node.js**, **HTML/CSS**, **Markdown**, **LaTex**<br></br>

Databases
:    **MySQL**, **MongoDB**<br></br>


Cloud
:    **AWS** EC2, S3, Batch, Lambda, RDS, EventBridge, StepFunctions **Google** BigQuery<br></br>

Other
:    **Jupyter**, **Git**, **Docker**, **Tableau**<br></br>




Selected Publications
----------

* Gregory P Way, **Ted Natoli,** Adeniyi Adeboye, et al. ["Morphology and gene expression profiling provide complementary information for mapping cell state"](https://www.biorxiv.org/content/10.1101/2021.10.21.465335v1) BioRxiv. October 22, 2021.

* Francesca Rapino, **Ted Natoli,** Erika Norabuena, Juhi Narula, Dane Hazelbaker, et al. 2021. "Small molecule screen reveals pathways that regulate C4 secretion in stem-cell derived astrocytes." *in submission*

* Blasco, Andrea, **Ted Natoli,** Michael G. Endres, Rinat A. Sergeev, Steven Randazzo, Jin H. Paik, N. J. Maximilian Macaluso, et al. n.d. ["Improving Deconvolution Methods in Biology through Open Innovation Competitions: An Application to the Connectivity Map."](https://doi.org/10.1093/bioinformatics/btab192) Bioinformatics. April 7, 2021.

* Jin, Xin, Zelalem Demere, Karthik Nair, Ahmed Ali, Gino B. Ferraro, **Ted Natoli**, Amy Deik, et al. 2020. ["A Metastasis Map of Human Cancer Cell Lines."](http://dx.doi.org/10.1038/s41586-020-2969-2) Nature 588 (7837): 331–36.

* Amy Li, Xiaodong Lu, **Ted Natoli,** Joshua Bittker, Nisha S. Sipes, Aravind Subramanian, Scott Auerbach, David H. Sherr, and Stefano Monti. 2019. ["The Carcinogenome Project: In Vitro Gene Expression Profiling of Chemical Perturbations to Predict Long-Term Carcinogenicity."](https://www.ncbi.nlm.nih.gov/pubmed/30964323) Environmental Health Perspectives 127 (4).

* Subramanian, Aravind, Rajiv Narayan, Steven M. Corsello, David D. Peck, **Ted E. Natoli,** Xiaodong Lu, Joshua Gould, et al. 2017. ["A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles."](https://www.ncbi.nlm.nih.gov/pubmed/29195078) Cell 171 (6): 1437–52.e17.

* Smith, Ian, Peyton G. Greenside, **Ted Natoli,** David L. Lahr, David Wadden, Itay Tirosh, Rajiv Narayan, et al. 2017. ["Evaluation of RNAi and CRISPR Technologies by Large-Scale Gene Expression Profiling in the Connectivity Map."](https://www.ncbi.nlm.nih.gov/pubmed/29190685) PLoS Biology 15 (11): e2003213.

<!-- * Blasco, Andrea, Michael G. Endres, Rinat A. Sergeev, Anup Jonchhe, Max Macaluso, Rajiv Narayan, **Ted Natoli,** et al. 2019. ["Advancing Computational Biology and Bioinformatics Research Through Open Innovation Competitions."](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0222165) PLOS ONE 14(9): e0222165. -->

<!-- * Litichevskiy, Lev, Ryan Peckner, Jennifer G. Abelin, Jacob K. Asiedu, Amanda L. Creech, John F. Davis, Desiree Davison, et al. 2018. ["A Library of Phosphoproteomic and Chromatin Signatures for Characterizing Cellular Responses to Drug Perturbations."](https://www.ncbi.nlm.nih.gov/pubmed/29655704) Cell Systems 6 (4): 424–43.e7. -->

<!-- * Berger, Alice H., Angela N. Brooks, Xiaoyun Wu, Yashaswi Shrestha, Candace Chouinard, Federica Piccioni, Mukta Bagul, et al. 2016. ["High-Throughput Phenotyping of Lung Cancer Somatic Mutations."](https://www.ncbi.nlm.nih.gov/pubmed/27478040) Cancer Cell 30 (2): 214–28. -->

<!-- * Li A, Lu X, Natoli T, Bittker J, Sipes NS, Subramanian A, Auerbach S, Sherr DH, Monti S. ["The Carcinogenome Project: In Vitro Gene Expression Profiling of Chemical Perturbations to Predict Long-Term Carcinogenicity."](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6785232/) Environ Health Perspect. 2019 Apr;127(4):47002. doi: 10.1289/EHP3986. PMID: 30964323; PMCID: PMC6785232. -->


----


